### DISCUSSION TITLE
Flutter: Localization naming conventions 🌍

### DISCUSSION THREAD

**Discussion by jackcolley** [netsells/code-standards#558] [Ideas]
Working on a big project with a lot of translatable strings can easily become a mess and hard to work with.

### The goal 📈
To improve the engineer experience & efficiency when working on multilingual projects

### The idea 💡
* Sort all translatable strings alphabetically by their key
```json
"loginScreenBody": "Enter your email address and password to sign in",
"loginScreenHeader": "Login"
```

* Use a consistent naming convention for all strings
`{screen name}{section}{item}`

**Comment by ptrbrynt** [comment]
Definitely a good idea!
